apps/dashboard/src/app/drops/[slug]/page.tsx

```diff
@@ -0,0 +1,134 @@
+
+export const DROP_PAGES: DropPageData[] = [
+  {
+    slug: "test",
+    type: "erc1155",
+    contractAddress: "0xBD9d7f15f3C850B35c30b8F9F698B511c20b7263",
+    tokenId: 0n,
+    chainId: 11155111,
+    hideQuantitySelector: true,
+    hideMintToCustomAddress: true,
+    thumbnail: "/drop/thumbnails/zero-x-thirdweb.mp4",
+  },
+  {
+    slug: "zero-chain-announcement",
+    type: "erc1155",
+    contractAddress: "0x78264a0af02d894f2d9ae3e11E4a503b352CC437",
+    tokenId: 0n,
+    chainId: 543210,
+    hideMintToCustomAddress: true,
+    hideQuantitySelector: true,
+    thumbnail: "/drop/thumbnails/zero-x-thirdweb.mp4",
+  },
+];
+
+export default async function DropPage({
+  params,
+}: { params: { slug: string } }) {
+  const { slug } = params;
+
+  const project = DROP_PAGES.find((p) => p.slug === slug);
+
+  if (!project) {
+    return notFound();
+  }
+  // eslint-disable-next-line no-restricted-syntax
+  const chain = defineDashboardChain(project.chainId, undefined);
+  const client = getThirdwebClient();
+
+  const contract = getContract({
+    address: project.contractAddress,
+    chain,
+    client,
+  });
+
+  const [nft, claimCondition, contractMetadata] = await Promise.all([
+    project.type === "erc1155"
+      ? getNFT1155({ contract, tokenId: project.tokenId })
+      : getNFT721({ contract, tokenId: 0n }),
+    project.type === "erc1155"
+      ? getActiveClaimCondition1155({
+          contract,
+          tokenId: project.tokenId,
+        }).catch(() => undefined)
+      : getActiveClaimCondition721({ contract }).catch(() => undefined),
+    getContractMetadata({ contract }),
+  ]);
+
+  const thumbnail =
+    project.thumbnail || nft.metadata.image || contractMetadata.image || "";
+
+  const displayName = contractMetadata.name || nft.metadata.name || "";
+
+  const description =
+    contractMetadata.description || nft.metadata.description || "";
+
+  if (!claimCondition) {
+    return (
+      <NftMint
+        contract={contract}
+        displayName={displayName || ""}
+        thumbnail={thumbnail}
+        description={description || ""}
+        {...project}
+        noActiveClaimCondition
+      />
+    );
+  }
+
+  const currencyMetadata = claimCondition.currency
+    ? await getCurrencyMetadata({
+        contract: getContract({
+          address: claimCondition.currency,
+          chain,
+          client,
+        }),
+      })
+    : undefined;
+
+  const currencySymbol = currencyMetadata?.symbol || "";
+  if (!currencyMetadata?.decimals) {
+    return notFound();
+  }
+  const pricePerToken = Number(
+    toTokens(claimCondition.pricePerToken, currencyMetadata.decimals),
+  );
+
+  return (
+    <NftMint
+      contract={contract}
+      displayName={displayName || ""}
+      thumbnail={thumbnail}
+      description={description || ""}
+      currencySymbol={currencySymbol}
+      pricePerToken={pricePerToken}
+      noActiveClaimCondition={false}
+      {...project}
+    />
+  );
+}
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +135]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Add page metadata - title, description, opengraph etc<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5307?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
updated